python/cuml/common/array_descriptor.py

```diff
@@ -54,8 +54,12 @@ class CumlArrayDescriptor():
     Python descriptor object to control getting/setting `CumlArray` attributes
     on `Base` objects. See the Estimator Guide for an in depth guide.
     """
+    def __init__(self, order='K'):
+        self.order = order
```

Comment:
Rather than handling this at the descriptor level, it is probably better to defer to the array itself.

Reply:
The problem is that the `CumlArrayDescriptor` attributes are usually initialized with `CumlArray.zeros` that sets an order during training. However, with CPU/GPU interop, we might now sometimes bypass training and directly set the attributes for inference. In order to perform the copy the numpy arrays incoming from the CPU estimator are transformed into `CumlArray`s. During this process, the order should be set to correspond to the order specified during the `CumlArray.zeros` initialization that we have no access to. [With this change in the Base class](https://github.com/rapidsai/cuml/blob/63fa6081f5563b8ad2c91dbf5a1bd31008085c76/python/cuml/experimental/common/base.pyx#L339-L349), the order is set correctly to what is expected by the C++ layer.